src/services/types.ts

```diff
@@ -130,3 +136,11 @@ export type ReferrerReward = {
   timestamp: number;
   amount: string;
 };
+
+export type ReferrerSetReferrer = {
+  referrer: string;
+};
+
+export type ReferrerReserve = {
+  amount: string;
```

Comment:
Could you define all attributes from `historyElement ``data `in this type please? 
https://github.com/sora-xor/sora-subquery/blob/develop/src/mappings/referralReserveHandler.ts#L16

```
from: string;
to: string;
amount: string;
```